cmd/cosign/cli/verify/verify.go

```diff
@@ -27,15 +27,19 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/sigstore/cosign/cmd/cosign/cli/fulcio"
+	"github.com/sigstore/cosign/cmd/cosign/cli/options"
+	"github.com/sigstore/cosign/cmd/cosign/cli/sign"
 	"github.com/sigstore/cosign/pkg/cosign"
 	"github.com/sigstore/cosign/pkg/cosign/pivkey"
+	"github.com/sigstore/cosign/pkg/image"
+	sigs "github.com/sigstore/cosign/pkg/signature"
 	"github.com/sigstore/sigstore/pkg/signature"
 	"github.com/sigstore/sigstore/pkg/signature/payload"
 )
 
 // VerifyCommand verifies a signature on a supplied container image
 type VerifyCommand struct {
```

Comment:
This one's weirder than the other surfaces, but would be good to address the linter feedback here, too

Reply:
yeah, agreed, but not in this PR?

Reply:
sgtm